src/apps/answers/domain/answers.py

```diff
@@ -639,3 +639,8 @@ class IdentifiersQueryParams(InternalModel):
     respondent_id: uuid.UUID | None = None
     target_subject_id: uuid.UUID | None = None
     answer_id: uuid.UUID | None = None
+
+
+class TakeNowValidationResponse(PublicModel):
+    valid: bool
+    message: str | None = None
```

Comment:
Can we use a response `code` instead of a `message` (or maybe both 🤷). It would allow me to translate the code to a user-friendly localised error message on the FE. I'm thinking these codes would be great to start:
- `invalid_applet_id`
- `no_access_to_applet`
- `invalid_activity_or_flow_id`
- `insufficient_role_permissions`
- `invalid_source_subject`
- `invalid_target_subject`

Reply:
Do we already have all possible error messages in designs? this message field in API in return is not to direct show in frontend

Reply:
The error messages aren't in the designs, but they were created on the FE based on the acceptance criteria in a previous ticket. So they are available

Reply:
done, but to keep API common behaviour, in case of applet not exists, will return http status 404 and in case of user not be a manager, will return 403